dart/lib/src/sentry.dart

```diff
@@ -219,4 +220,35 @@ class Sentry {
 
     return true;
   }
+
+  static ISentrySpan startTransaction(
+    String name,
+    String operation, {
+    String? description,
+    bool? bindToScope,
+  }) =>
+      _hub.startTransaction(
+        name,
+        operation,
+        description: description,
+        bindToScope: bindToScope,
+      );
+
+  static ISentrySpan startTransactionWithContext(
+    SentryTransactionContext transactionContext, {
+    Map<String, dynamic>? customSamplingContext,
+    bool? bindToScope,
+  }) =>
+      _hub.startTransactionWithContext(
+        transactionContext,
+        customSamplingContext: customSamplingContext,
+        bindToScope: bindToScope,
+      );
+
+  // missing traceHeaders
+
+  ISentrySpan? getSpan() => _hub.getSpan();
+
+  /// internal method
+  static Hub get currentHub => _hub;
```

Comment:
had to add it back because of the HubAdapter, we have to call `hub.captureTransaction` so either we add `captureTransaction` to the Sentry static class as internal or add back the internal `currentHub`

Reply:
keep `currentHub` and make it internal